src/server/req_manager.c

```diff
@@ -1846,22 +1842,6 @@ mgr_sched_unset(struct batch_request *preq)
 				svr_save_db(&server, SVR_SAVE_FULL);
 				free_svrattrl(t_list);
 			}
-		} else if (strcasecmp(tmp_plist->al_name,
-				ATTR_sched_preempt_queue_prio) == 0) {
-			unset_sched_preempt_queue_prio(&psched->sch_attr[(int) SCHED_ATR_preempt_queue_prio], psched->sc_name);
-			set_scheduler_flag(SCH_ATTRS_CONFIGURE, psched);
-		} else if (strcasecmp(tmp_plist->al_name,
-				ATTR_sched_preempt_prio) == 0) {
-			unset_sched_preempt_prio(&psched->sch_attr[(int) SCHED_ATR_preempt_prio], psched->sc_name);
-			set_scheduler_flag(SCH_ATTRS_CONFIGURE, psched);
-		} else if (strcasecmp(tmp_plist->al_name,
-				ATTR_sched_preempt_order) == 0) {
-			unset_sched_preempt_order(&psched->sch_attr[(int) SCHED_ATR_preempt_order], psched->sc_name);
-			set_scheduler_flag(SCH_ATTRS_CONFIGURE, psched);
-		} else if (strcasecmp(tmp_plist->al_name,
-				ATTR_sched_preempt_sort) == 0) {
-			unset_sched_preempt_sort(&psched->sch_attr[(int) SCHED_ATR_preempt_sort], psched->sc_name);
```

Comment:
I thought functions like unset_sched_preempt_sort() (and the others) reverted the attributes back to their default values when unset.  Don't we still need to do that?

Reply:
for scheduler attributes that is done in set_sched_default() called below, so we do not need the unset functions.

Reply:
Gotcha, sounds good.